suche.php

```diff
@@ -234,6 +234,25 @@ function process_notify_items(&$items) {
         $end = '</div><!-- ende Inhalt -->';
         $body = substr($body, strpos($body, $start));
         $body = substr($body, 0, strpos($body, $end) + strlen($end));
+        mkdir($details_dir.'/'.$id);
+        preg_match_all('!href="(\?[^>]+)"/>\W*(.*)</!m', $body, $matches, PREG_SET_ORDER);
+        foreach ($matches as $match) {
+            $url = '/index.php' . $match[1];
+            $filename = $match[2];
+            $filename = preg_replace('![^_a-zA-Z0-9\.]!m', '', $filename);
+
+            $response = $client->get($url, ['headers' => $headers]);
+            if ($response->getStatusCode() != 200) {
+                echo 'Error: fetch file '.$url.' - '.$response->getStatusCode();
+                continue;
+            }
+
+            file_put_contents($details_dir.'/'.$id . '/' . $filename, $response->getBody()->getContents());
```

Comment:
proper cleanup is not possible, because this ID can't be determined from the canceled ones. 

`0023 K 0283/2018` vs `35582`

Reply:
🤷